portfolio/src/main/webapp/script.js

```diff
@@ -12,14 +12,136 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let map;
+const locationsDict = new Object();
+
+class Location {
+  constructor(marker, lat, lng, header, pitch, infoString, infoWindow) {
+    this.marker = marker;
+    this.lat = lat;
+    this.lng = lng;
+    this.header = header;
+    this.pitch = pitch;
+    this.infoString = infoString;
+    this.infoWindow = infoWindow;
+  }
+}
+
+/** Calls all necessary functions to be called onLoad */
+function initialize() {
+  loadComments();
+  createMap();
+}
+
+/** Creates a map with all components and adds it to the page. */
+function createMap() {
+  map = new google.maps.Map(document.getElementById("map"), {
+    center: { lat: 41.8996099, lng: -87.6232902, h: -7, p: 22 },
+    zoom: 13,
+  });
+
+  setupMarkers();
+  changeView(locationsDict["Bean"]);
+}
+
+/** All setup code for markers. */
+function setupMarkers() {
+  locationsDict["Bean"] = new Location(
+    null,
+    41.8826099,
+    -87.6232902,
+    -7,
+    22,
+    '<p style="margin-top:-3px; margin-bottom:-3px" class="field-text">Cloud Gate</p>',
+    null
+  );
+  locationsDict["Skyline"] = new Location(
+    null,
+    41.9147471,
+    -87.6209924,
+    185,
+    17,
+    '<p style="margin-top:-3px; margin-bottom:-3px" class="field-text">Photography Point</p>',
+    null
+  );
+  locationsDict["Riverwalk"] = new Location(
+    null,
+    41.8877563,
+    -87.6273952,
+    125,
+    30,
+    '<p style="margin-top:-3px; margin-bottom:-3px" class="field-text">Chicago Riverwalk</p>',
+    null
+  );
```

Comment:
I mentioned this in the last review, but if these are static does it make sense to define these as global const variables.